optaplanner-docs/src/modules/ROOT/pages/_attributes.adoc

```diff
@@ -3,4 +3,8 @@
 :quickstarts-archive-url: https://www.optaplanner.org/download/download.html
 :hello-world-java-quickstart-url: https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/hello-world
 :spring-boot-quickstart-url: https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/technology/java-spring-boot
-:quarkus-quickstart-url: https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/school-timetabling
\ No newline at end of file
+:quarkus-quickstart-url: https://github.com/kiegroup/optaplanner-quickstarts/tree/stable/use-cases/school-timetabling
+:optaplanner: OptaPlanner
+:quarkus: Quarkus
+:optaplanner-jdk-version: 11
```

Comment:
The java version is already coming in through the maven/antora build IIRC. @yurloc Can you verify (or check with Radovan)?